src/Elastic.Apm/DiagnosticListeners/HttpDiagnosticListener.cs

```diff
@@ -32,7 +32,7 @@ internal class HttpDiagnosticListener : IDiagnosticListener
 
 		private AbstractLogger Logger { get; }
 
-		public string Name => "HttpHandlerDiagnosticListener";
```

Comment:
This should be reverted. Otherwise it'll break our existing HttpClient monitoring stuff. I'd simply leave it as it is currently on master (so subscribe to this and move on, although as we saw this won't work on Full Framework due to different keys and types in the DiagnosticSource events.) Then in a later commit we can address `HttpClient` monitoring on Full Framework.

Reply:
I agree. So for now I should also remove call to Agent.Subscribe(new HttpDiagnosticsSubscriber()) in Init since it won't do anything anyway. And I'll open an issue to support monitoring HttpClient for Full Framework. How does it sound?

Reply:
👍 Perfect, sounds good! If you want I can take that issue, I already looked into this and have something in mind to solve this.

Reply:
gr8 - here is the issue "Add support for tracing calls via HttpClient on Full Framework #115"